src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -25,8 +43,33 @@ export class HotbarMenu extends React.Component<Props> {
     return hotbar.items.map((item) => catalogEntityRegistry.items.find((entity) => entity.metadata.uid === item.entity.uid)).filter(Boolean);
   }
 
+  previous() {
```

Comment:
Maybe better to move logic from `previous()` and `next()` methods into store itself. To keep clean `HotbarMenu` from store internals.

Reply:
Refactored.

Reply:
Probably don't need them to be part of this component then.